src/main/java/uk/gov/companieshouse/itemhandler/kafka/OrdersKafkaConsumer.java

```diff
@@ -228,9 +226,8 @@ protected void republishMessageToTopic(String orderUri, String currentTopic, Str
         try {
             kafkaProducer.sendMessage(createRetryMessage(orderUri, nextTopic));
         } catch (ExecutionException | InterruptedException e) {
-            logMap.put(LoggingUtils.EXCEPTION, e);
             LoggingUtils.getLogger().error(String.format("Error sending message: \"%1$s\" to topic: \"%2$s\"",
-                    orderUri, nextTopic), logMap);
+                    orderUri, nextTopic), e, logMap);
             if (e instanceof InterruptedException) {
```

Comment:
Shouldn't this thread interruption be the last operation in the catch block (after logging)?

Reply:
It is the last operation in the catch block?
`try {
            kafkaProducer.sendMessage(createRetryMessage(orderUri, nextTopic));
        } catch (ExecutionException | InterruptedException e) {
            LoggingUtils.getLogger().error(String.format("Error sending message: \"%1$s\" to topic: \"%2$s\"",
                    orderUri, nextTopic), e, logMap);
            if (e instanceof InterruptedException) {
                Thread.currentThread().interrupt();
            }
        }`

Reply:
Yes, you are right. Didn't notice hidden lines. Thanks